src/consensus/params.h

```diff
@@ -11,6 +11,8 @@
 #include <map>
 #include <string>
 
+#include <script/script.h> // mandatory_coinbase_destination
```

Comment:
Isn't this dependency giving you problems? I would swear I tried the same with https://github.com/ElementsProject/elements/pull/433

Reply:
hopefully not, we do the same thing for elements-0.14.1 IIRC!